pkg/alertmanager/state_replication_test.go

```diff
@@ -50,6 +65,25 @@ func (f *fakeReplicator) GetPositionForUser(_ string) int {
 	return 0
 }
 
+func (f *fakeReplicator) ReadFullStateForUser(ctx context.Context, userID string) ([]*clusterpb.FullState, error) {
+	if f.readNum >= len(f.reads) {
+		return nil, nil
```

Comment:
Should we return error given it's an unexpected condition in tests (eg. test fixture has not been mocked correctly)?